.github/workflows/ci-testing.yml

```diff
@@ -148,20 +148,20 @@ jobs:
         FIFTYONE_DO_NOT_TRACK: true
       run: |
         coverage run --source flash -m pytest flash tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
-        coverage xml
 
     - name: Upload pytest test results
       uses: actions/upload-artifact@v2
       with:
         name: pytest-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}
         path: junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
       # Use always() to always run this step to publish test results when there are test failures
-      if: failure()
+      if: always()
```

Comment:
I do not think it is needed to upload test report when everything is fine :P

Reply:
Fixed :smiley: